spec/models/comment_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
```

Comment:
- [ ] As mentioned by the previous reviewer 17 out of 24 tests are failing. Please fix them

![image](https://user-images.githubusercontent.com/83803414/178113009-4701b3b7-65c5-4fe2-ac44-908e9e4ca6af.png)

Reply:
Hi! on my PC my tests pass. I have made no changes and this is my output:
![Captura de pantalla de 2022-07-09 10-52-41](https://user-images.githubusercontent.com/73128809/178113395-109efa17-4e6d-4620-8ac3-ac2784d5561f.png)
Otherwise, maybe a package was missing or something since all the installs weren't completed beforehand. The tests passed as well.

Can you try this and let me know if this doesn't work? The logic should be passing, but I think that maybe the setup on your PC wasn't complete

Reply:
Please be sure to complete the following steps in your terminal while within the root directory for this project. These must be run before running the tests

- `git checkout validation_and_specs`
- `bundle install`
- `rails db:create`
- `rspec`